plugins/modules/remote_management/ipmi/ipmi_power.py

```diff
@@ -51,13 +51,30 @@
             - reset -- Request system reset without waiting for OS
             - boot -- If system is off, then 'on', else 'reset'"
     choices: ['on', 'off', shutdown, reset, boot]
-    required: true
     type: str
   timeout:
     description:
       - Maximum number of seconds before interrupt request.
     default: 300
     type: int
+  machine:
+    description:
+      - Provide a list of the remote target address for the bridge IPMI request,
+        and the power status.
+    required: false
+    type: list
+    elements: dict
+    suboptions:
+      targetAddress:
+        description:
+          - Remote target address for the bridge IPMI request.
+        type: int
```

Comment:
From the code it seems that this is required.
```suggestion
        type: int
        required: true
```